taichi/backends/cpu/cpu_device.cpp

```diff
@@ -26,14 +27,12 @@ DeviceAllocation CpuDevice::allocate_memory(const AllocParams &params) {
   return alloc;
 }
 
-DeviceAllocation CpuDevice::allocate_memory_runtime(const AllocParams &params,
-                                                    JITModule *runtime_jit,
-                                                    LLVMRuntime *runtime,
-                                                    uint64 *result_buffer) {
+DeviceAllocation CpuDevice::allocate_memory_runtime(
+    AllocParamsLlvmRuntime &params) {
   AllocInfo info;
-  info.ptr = allocate_llvm_runtime_memory_jit(runtime_jit, runtime, params.size,
-                                              result_buffer);
+  info.ptr = allocate_llvm_runtime_memory_jit(params);
   info.size = params.size;
+  info.use_cached = params.use_cached;  // TODO add cpu caching allocator
```

Comment:
Nit for here and below: The widely used format is `// TODO: ...`, and is written on a separate line